README.md

```diff
@@ -789,14 +789,13 @@ const reduceRightArray = arr.reduceRight((acc, current) => {
 **[⬆ Back to Top](#table-of-contents)**
 ### Abort Fetch
 
-```javascript
-
-
-//HTML
+```html
+//html
 <button id="download">Download</button>
-<button id="abort">Abort</button>
+ <button id="abort">Abort</button>
```

Comment:
I think you should remove the redundant space for aligning.

Reply:
Yeah. sure.